CHANGELOG.md

```diff
@@ -63,6 +63,7 @@ All notable changes to this project will be documented in this file.
         * `conditional_jump_triggers` and `pattern_opcode_events` repeated capabilities - [#1191](https://github.com/ni/nimi-python/issues/1191), [#1192](https://github.com/ni/nimi-python/issues/1192)
     * #### Changed
         * `write_source_waveform_site_unique()` now supports `numpy.array` and `list` as site waveform types
+        * `send_software_edge_trigger()` no longer takes any parameters. To send a SW start trigger, use the session `session.send_software_edge_trigger()`. To send a Conditional Jump Trigger, use the repeated capability `session.conditional_jump_triggers[1].send_software_edge_trigger()` - [#1193](https://github.com/ni/nimi-python/issues/1193)
```

Comment:
How will this design scale when we add a new single-instance trigger type?

Reply:
The code would need to look at what the repeated capability string begins with and select the trigger type that way.

Reply:
Hmm, so code would look like:
```
session.shiny_new_trigger[0].send_software_edge_trigger()
```
..even if the driver supports only one instance of shiny_new_trigger?

Reply:
Yes, and `shiny_new_trigger` would need to have a unique string in the repeated capabilities, like `"conditionalJumpTrigger"`. Unless you know of another way to differentiate the different trigger types.

Reply:
#1220 for discussing this